cms/migrations/0010_migrate_use_structure.py

```diff
@@ -14,31 +15,33 @@ def forwards(apps, schema_editor):
     try:
         ph_ctype = ContentType.objects.get_for_model(ph_model)
         page_ctype = ContentType.objects.get_for_model(page_model)
-        permission, _ = Permission.objects.get_or_create(
+        permission, __ = Permission.objects.get_or_create(
             codename='use_structure', content_type=ph_ctype, name=u"Can use Structure mode")
```

Comment:
As `__future__.unicode_literals` is imported, shouldn't `u'strings'` be replaced by `'strings'` ?
Else, LGTM to be merged.